README.md

```diff
@@ -141,10 +153,19 @@ There are a few main concepts that should help you get started:
 
 ### References & Useful Resources
 
-* [builder command - go.opentelemetry.io/collector/cmd/builder - Go Packages](https://pkg.go.dev/go.opentelemetry.io/collector/cmd/builder#section-readme)
-* [Building a Trace Receiver](https://opentelemetry.io/docs/collector/trace-receiver/#representing-operations-with-spans)
-* [Building a custom collector](https://opentelemetry.io/docs/collector/custom-collector/)
-* [otel4devs/builder-config.yaml at main · rquedas/otel4devs](https://github.com/rquedas/otel4devs/blob/main/collector/receiver/trace-receiver/builder-config.yaml)
-* [opentelemetry-collector-contrib/receiver/activedirectorydsreceiver at main · open-telemetry/opentelemetry-collector-contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/activedirectorydsreceiver)
-* [opentelemetry-collector-contrib/extension/basicauthextension at main · open-telemetry/opentelemetry-collector-contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/basicauthextension)
-
+* [builder command - go.opentelemetry.io/collector/cmd/builder - Go Packages][3]
```

Comment:
I think in the future, these resources might be migrated, or elaborated on in a contributing doc OR in the o11y docs sites. Just making the mention. No action needed.